concepticondata/conceptlists/Pagel-2018-200.tsv

```diff
@@ -0,0 +1,201 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	RATE
+Pagel-2018-200-1	1	all	98	ALL	0.000185864
+Pagel-2018-200-2	2	and	1577	AND	0.000170797
+Pagel-2018-200-3	3	animal	619	ANIMAL	0.00019532
+Pagel-2018-200-4	4	ashes	646	ASH	0.00022033
+Pagel-2018-200-5	5	at	1461	AT	0.000316527
+Pagel-2018-200-6	6	back			0.000279398
```

Comment:
this is oviously the spine, that is BACK in concepticon, we have tthis concept.

Reply:
Was not a question of whether the concept is available but rather: back (behind), back (of an animal), back (in time), back (in direction).

Reply:
obviously the one to which both IELex and ABVD and the Bantu list link (I suppose it's the body part)